crm_opportunity_currency/README.rst

```diff
@@ -0,0 +1,77 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
pls use new readme structure for new modules

Reply:
Changed